guides/release/typescript/index.md

```diff
@@ -0,0 +1,43 @@
+This guide is designed to help you get up and running with TypeScript in an Ember app.
+
+This is _not_ an introduction to TypeScript _or_ Ember. Throughout this guide, we’ll link back to [the TypeScript docs][typescript-docs] and [the Ember Guides][ember-guides] when there are specific concepts that we will not explain here but which are important for understanding what we’re covering!
```

Comment:
This makes it sound like we are linking to the Ember Guides as a separate resource, but I think we're actually making this a part of the Ember Guides. Maybe:

```suggestion
This is _not_ an introduction to TypeScript _or_ Ember. Throughout this guide, we’ll link back to [the TypeScript docs][typescript-docs] and to other sections of [the Ember Guides][ember-guides] when there are specific concepts that we will not explain here but which are important for understanding what we’re covering!
```

Reply:
Heh, that's because this content is all originally pulled from (though *massively* updated, thanks to @gitKrystan’s amazing efforts) https://docs.ember-cli-typescript.com (which can go away/redirect here once this lands!). So originally it *was* just pointing the the Ember Guides as a separate resource. 😅